src/Microsoft.Framework.Runtime.Roslyn/RoslynProjectReference.cs

```diff
@@ -71,7 +63,7 @@ public Assembly Load(IAssemblyLoadContext loadContext)
             using (var pdbStream = new MemoryStream())
             using (var assemblyStream = new MemoryStream())
             {
-                IList<ResourceDescription> resources = CompilationContext.Resources;
+                IList<ResourceDescription> resources = BeforeCompileContext.Resources;
```

Comment:
I really find this is clearer than `var`. With that being said, I'll change it to `var`. :cry: